scripts/utils.js

```diff
@@ -0,0 +1,47 @@
+/* jshint node: true */
+'use strict';
+
+const spawn = require('cross-spawn');
+
+function dirHasChanges(dir) {
+  return exec('git', ['status', dir, '--porcelain'])
+    .then((result) => {
+      // Untracked files are prefixed with '??'
+      // https://git-scm.com/docs/git-status/1.8.1#_output
+      // https://stackoverflow.com/a/6978402/6178885
+      return !!(result && result.indexOf('??') > -1);
```

Comment:
What about files that are already tracked but modified?

Reply:
Since we're only dealing with images, they can only be new, unmodified files. Unless I'm missing a specific scenario?